packages/ui/src/DataTable/components/Actions/index.tsx

```diff
@@ -1,19 +1,24 @@
 //import { useRouter } from 'next/router';
 import { ButtonProps } from "primereact/button";
 import { MenuItem } from "primereact/menuitem";
-import React from "react";
+import React, { useState } from "react";
 
-import { Menu } from "../../../Menu";
+import { ConfirmationModal } from "@/ConfirmationModal";
+import { Menu } from "@/Menu";
 
 export interface ActionsMenuProperties {
   actions?: MenuItem[];
   buttonOptions?: Omit<ButtonProps, "onClick">;
+  data?: object;
   deleteIcon?: string;
   deleteLabel?: string;
   editIcon?: string;
   editLabel?: string;
+  header?: string;
+  message?: string;
```

Comment:
name should indicate what they are for:
```suggestion
  deleteConfirmationHeader?: string;
  deleteConfirmationMessage?: string;
```